nilearn/decomposition/dict_learning.py

```diff
@@ -222,20 +232,18 @@ def fit(self, imgs, y=None, confounds=None):
         if self.verbose:
             print('[DictLearning] Learning initial components')
         self._init_dict(data)
-
         self._raw_fit(data)
+        return self
```

Comment:
Maybe add a "Returns" section explaining that `fit` chaining is allowed. Do we really want this ?